controllers/pcs_handler.go

```diff
@@ -0,0 +1,152 @@
+package controllers
+
+import (
+	"encoding/xml"
+	"fmt"
+	appv1alpha1 "github.com/mshitrit/hasno-setup-operator/api/v1alpha1"
+	"os/exec"
+	"strings"
+)
+
+const (
+	podNameHALayer           = "peer"
+	containerNamePacemaker   = "pacemaker"
+	pcsDeploymentIdentifier  = "k8sDeployment"
+	pcsFenceIdentifier       = "stonith"
+	fenceAgentPrefix         = "fencing_agent_"
+	deploymentResourcePrefix = "pcs_deployment_resource_"
+)
+
+//TODO mshitrit maybe define a different component  ?
+type haStatus struct {
+	XMLName   xml.Name  `xml:"crm_mon"`
```

Comment:
Is `xml`: a thing?

Reply:
I'm using [encoding/xml](https://pkg.go.dev/encoding/xml) pkg in order to parse the the `pcs status xml` output.
The `xml` tag is used in order to identify which fields to be parsed into the struct.